### PR TITLE
test_oop Ruby

### PR DESCRIPTION
**Pull request by Vali-Mandeal**

### CODE REVIEW

**Comment by alexandru-calinoiu** [comment]
Please give a proper name to the pr and add a description to it with a minimal info on how to run the code, ruby version other deps ....

**Review by alexandru-calinoiu — CHANGES_REQUESTED** [review]
Please give a proper name to the pr and add a description to it with a minimal info on how to run the code, ruby version other deps ....

**Comment by Vali-Mandeal** [comment]
Added proper description

File: Main.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+dirPath = File.dirname(File.dirname(File.expand_path(__FILE__)))
+load dirPath + "/CoffeeShopApp/Objects/CoffeeShop.rb"
```
Comment (alexandru-calinoiu) [inline Main.rb:2]: When I try to run the code via

```
ruby Main.rb
```

I get the following error

Traceback (most recent call last):
        1: from Main.rb:2:in `<main>'
Main.rb:2:in `load': cannot load such file -- /<path>/CoffeeShopApp/Objects/CoffeeShop.rb (LoadError)
```

Please take a look at [require](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-require) and [require_relative](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-require_relative) kernel methods.

**Review by alexandru-calinoiu — CHANGES_REQUESTED** [review]
Still very hard to review:

- Instruction on how to run should be in PR description not in a commit.
- See the comm that I left in the poorly named `Main.rb`

Every language has it's own idioms and coding styles, you have seemed to ignored them and this makes reviewing this impossible.

Please take a look at the https://www.rubocop.org/en/stable/ and focus on proper naming including filenames and variables.

**Comment by alexandru-calinoiu** [comment]
Closing this for now, will followup via email.

**Pull request closed without merging**